src/test/java/net/sf/jabref/importer/fileformat/MedlinePlainImporterTest.java

```diff
@@ -0,0 +1,120 @@
+
+        Assert.assertFalse(Testimporter
+                .isRecognizedFormat(MedlinePlainImporter.class.getResourceAsStream("IsiImporterTestMedline.isi")));
+
+        Assert.assertFalse(Testimporter.isRecognizedFormat(
+                MedlinePlainImporter.class.getResourceAsStream("MedlinePlainImporterTestEmpty.txt")));
+
+        //Test with txt File who is in recognized Format
+        Assert.assertTrue(Testimporter
+                .isRecognizedFormat(MedlinePlainImporter.class.getResourceAsStream("MedlinePlainImporterTest.txt")));
+
+
+
+    }
+
+    @Test
+    public void testImportEntries() throws Exception {
+
+        MedlinePlainImporter Testimporter = new MedlinePlainImporter();
+
+        //create entries list from the txt file
+        List<BibtexEntry> entries = Testimporter.importEntries(
+                MedlinePlainImporter.class.getResourceAsStream("MedlinePLainImporterTest.txt"),
+                new OutputPrinterToNull());
+        Assert.assertEquals(7, entries.size());
+
+        //create Bibtexentry from first/second object in the entry list
+        //and test with getField
+        BibtexEntry test1 = entries.get(0);
+        Assert.assertNull(test1.getField("month"));
+        Assert.assertEquals("Long, Vicky and Marland, Hilary", test1.getField("author"));
+        Assert.assertEquals(
+                "From danger and motherhood to health and beauty: health advice for the factory girl in early twentieth-century Britain.",
+                test1.getField("title"));
+        BibtexEntry test2 = entries.get(1);
+        Assert.assertEquals("06", test2.getField("month"));
+        Assert.assertNull(test2.getField("author"));
+        Assert.assertNull(test2.getField("title"));
+
+        //Test entries with empty txt File
+        List<BibtexEntry> emptyEntries = Testimporter.importEntries(
+                MedlinePlainImporter.class.getResourceAsStream("MedlinePLainImporterTestEmpty.txt"),
+                new OutputPrinterToNull());
+        Assert.assertEquals(0, emptyEntries.size());
+
+    }
+
+    @Test
+    public void testGetFormatName() {
+        MedlinePlainImporter Testimporter = new MedlinePlainImporter();
+        String expected = Testimporter.getFormatName();
+        String actual = "";
+        Assert.assertFalse(expected.equals(actual));
+        actual = "medlinePLain";
+        Assert.assertFalse(expected.equals(actual));
+        actual = "MedlinePlain";
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testGetCLIId() {
+        MedlinePlainImporter Testimporter = new MedlinePlainImporter();
+        String expected = Testimporter.getCLIId();
+        String actual = "";
+        Assert.assertFalse(expected.equals(actual));
+        actual = "MedlinePLain";
+        Assert.assertFalse(expected.equals(actual));
+        actual = "medlineplain";
+        Assert.assertTrue(expected.equals(actual));
```

Comment:
Why not use `Assert.assertEquals`? Expected and actual must be switched, as expected is the fixed output String you know is correct whereas actual is the result of the method under test. 

`Assert.assertEquals("medlineplain", importer.getCLIId());`

What is more, variables must always start with a lower case letter and camel case (except when they are constants).